### PR TITLE
Disable non-material zoom buttons and enable double tap to zoom

### LINKED ISSUE #67: We need to be able to zoom out farther
I want to be able to zoom further out, but the button is greyed out. I know, on the github website, it can't zoom out any further either, but I want to be able to on the app, at least.

### LINKED ISSUE #231: Can't scroll gist left and right
Scrolling a gist left and right doesn't work. Sometimes it makes the app crash.

### PR DESCRIPTION
**Pull request by NBonaparte**
I removed the zoom in/out buttons, which are not material design (also requested in #214 No. 6), and enabled the wide viewport to allow double tapping to zoom. This also coincidentally fixes #231, and might also fix #67.

### CODE REVIEW

**Comment by jonan** [comment]
I've tried something similar a couple of times and I faced some side effects that made me revert the changes. I'll give your code a try and get back to you.

Thanks!

**Comment by jonan** [comment]
Markdown files are less readable after applying this patch

# Before
readme

# After
readme2

We might be able to use this only in code files.

**Comment by NBonaparte** [comment]
I've fixed it so `WideViewPort` is only used with non-markdown sources.

File: app/src/main/java/com/github/mobile/util/SourceEditor.java [hunk 1]
```diff
@@ -166,11 +167,14 @@ public SourceEditor setSource(final String name, final String content, final boo
     }
 
     private void loadSource() {
-        if (name != null && content != null)
+        if (name != null && content != null) {
             if (markdown)
                 view.loadDataWithBaseURL(null, content, "text/html", CHARSET_UTF8, null);
-            else
+            else {
+                view.getSettings().setUseWideViewPort(true);
```
Comment (jonan) [inline app/src/main/java/com/github/mobile/util/SourceEditor.java]: This doesn't work if you change the "show markdown" option. Better do something like this:
```
            if (markdown) {
                view.loadDataWithBaseURL(null, content, "text/html", CHARSET_UTF8, null);
                view.getSettings().setUseWideViewPort(false);
            } else {
                view.loadUrl(URL_PAGE);
                view.getSettings().setUseWideViewPort(true);
            }
```

Comment (NBonaparte) [inline app/src/main/java/com/github/mobile/util/SourceEditor.java]: Forgot about the toggle, sorry. Fixed now.

**Comment by jonan** [comment]
I've done quite a bit of testing and I'm quite happy with the result.
Thanks!

**Pull request closed without merging**